src/index.js

```diff
@@ -51,76 +55,16 @@ const pizzaData = [
 
 // Default app component
 
-function App() {
+const App = () => {
   // const c = console.log("");
   return (
     <div className="container">
       <Header />
-      <Menu />
+      <Menu pizzaData={pizzaData} />
       <Footer />
     </div>
   );
```

Comment:
create a separate file for App component too, and we shouild not export default components